app/views/govuk_publishing_components/components/_subscription-links.html.erb

```diff
@@ -1,21 +1,50 @@
 <%
   email_signup_link ||= false
+  email_signup_link_text ||= "Get email alerts"
   feed_link ||= false
+  feed_link_text ||= "Subscribe to feed"
+  feed_link_box_value ||= false
+  feed_link_data = false
+
+  if feed_link_box_value
+    feed_link = "#"
+    feed_link_data = {
+      controls: "feed-reader",
+      expanded: "false"
+    }
+  end
 %>
-<% if email_signup_link || feed_link %>
-  <section class="gem-c-subscription-links">
+<% if email_signup_link || feed_link || feed_link_box_value %>
+  <section class="gem-c-subscription-links" data-module="gem-toggle">
     <h2 class="visuallyhidden">Subscriptions</h2>
     <ul class="gem-c-subscription-links__list">
       <% if email_signup_link.present? %>
         <li class="gem-c-subscription-links__list-item">
-          <%= link_to "Get email alerts", email_signup_link, class: "gem-c-subscription-links__link gem-c-subscription-links__link--email-alerts" %>
+          <%= link_to email_signup_link_text, email_signup_link, class: "gem-c-subscription-links__link gem-c-subscription-links__link--email-alerts" %>
         </li>
       <% end %>
-      <% if feed_link.present? %>
+
+      <% if feed_link_box_value || feed_link.present? %>
         <li class="gem-c-subscription-links__list-item">
-          <%= link_to "Subscribe to feed", feed_link, class: "gem-c-subscription-links__link gem-c-subscription-links__link--feed" %>
+          <%= link_to feed_link_text, feed_link,
+              class: "gem-c-subscription-links__link gem-c-subscription-links__link--feed",
+              data: feed_link_data
+          %>
         </li>
       <% end %>
     </ul>
+
+    <% if feed_link_box_value %>
+      <div class="gem-c-subscription-links__feed-box js-hidden" id="feed-reader">
+        <p class="gem-c-subscription-links__feed-description js-hidden"><%= feed_link_text %></p>
```

Comment:
It looks like the `js-hidden` means that this is never shown? 

See this example in the component guide: https://govuk-publishing-compon-pr-294.herokuapp.com/component-guide/subscription-links/with_copyable_feed_link/preview

Reply:
If Javascript is disabled, it gets shown and the main feed link gets hidden. Seemed like a sensible way to keep the text but remove the link/button.